src/main/resources/db/migration/V5__Create_jobs_table.sql

```diff
@@ -0,0 +1,9 @@
+-- V1__Create_jobs_table.sql
+CREATE TABLE jobs (
```

Comment:
You need to update the version of your migration file to a much later version based on the version in the database. You cannot user v5 cus v5 already exist